packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import SectionHeader from 'terra-section-header';
+
+<SectionHeader
+  title="Section Header Example"
```

Comment:
Can you add an example of the level prop?

Reply:
Done. c73f8ac